core/modules/file/views/views_handler_field_file_icon.inc

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * @file
+ * Definition of views_handler_field_file_icon.
+ */
+
+/**
+ * @ingroup views_field_handlers
+ */
+class views_handler_field_file_icon extends views_handler_field {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function option_definition() {
+    $options = parent::option_definition();
+    $options['icon_size'] = array('default' => 32);
+    $options['display_type'] = array('default' => 'svg');
+    $options['add_alt_text'] = array('default' => TRUE);
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function options_form(&$form, &$form_state) {
+    parent::options_form($form, $form_state);
+    $form['icon_size'] = array(
+      '#type' => 'number',
+      '#title' => t('Icon size'),
+      '#min' => 8,
+      '#max' => 300,
+      '#default_value' => $this->options['icon_size'],
+      '#field_suffix' => 'px',
+    );
+    $form['display_type'] = array(
+      '#type' => 'radios',
+      '#title' => t('Display type'),
+      '#options' => array(
+        'svg' => t('Inline <code>&lt;svg&gt;</code> tag'),
+        'img' => t('Regular <code>&lt;img&gt;</code> tag'),
```

Comment:
These are awful strings for translators.
Something I actually tried to avoid. :grimacing: Can we please discuss that?
Because my previous strings were a result of discussion.

Reply:
Oops, I'm sorry @indigoxela. We can revert these. I think the added descriptions are probably sufficient anyway.

Reply:
Thank you for the feedback, and my apologies for changing a previously discussed string. I reverted this to what it was before.